proj/lagom.conf

```diff
@@ -1,12 +1,14 @@
-// https://github.com/lagom/lagom.git#1.5.x
+// https://github.com/lagom/lagom.git#master
+
+// perhaps eventually there will be a 1.6.x branch for us to track;
+// at present (November 2019) 1.6.x development is happening on master
```

Comment:
Cutting 1.6.x (and 2.8.x for Play) tomorrow.

Reply:
for now, in the playframework repo should I be using 2.7.x, or master?

Reply:
master for Akka 2.6